app/template_email_files/rest.py

```diff
@@ -72,3 +73,15 @@ def update_template_email_files(template_email_files_id, service_id, template_id
     dao_update_template_email_files(update_dict)
     return jsonify(data=template_email_files_schema.dump(update_dict)), 200
 
+
+@template_email_files_blueprint.route("/<uuid:template_email_files_id>/archive", methods=["POST"])
+def archive_template_email_files(template_email_files_id, template_id, service_id):
+    current_data = TemplateEmailFile.query.get_or_404(template_email_files_id)
```

Comment:
we do this fetch differently for update and archive endpoints - is there a reason for this?